### PR TITLE
docs: copy operator installation notes from 2.1 to 2.2

### PR DESCRIPTION
**Pull request by cbuto**
## Description of changes being made

Copying the changes made for 2.1 in https://github.com/mesosphere/dcos-docs-site/pull/4016/files to 2.2. I also fixed some bad links.

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by gracedo — APPROVED** [review]

**Review by cwyl02 — APPROVED** [review]

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ After you finish the installation, see [Spark Operator custom resource documenta
 
 -   Using CLI
 
-    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration) for details.
+    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration-using-the-cli) for details.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md:39]: I mean, I don't need to make a suggestion here, but we shouldn't change this file. This file can get changed in the `main` branch if it needs to get updated (`develop` branch should just be for `/2.2/` version changes)

```suggestion
    See [Application Deployment](../../application-deployment#deploy-an-application-with-a-custom-configuration) for details.
```

**Review by msdolbey — CHANGES_REQUESTED** [review]
We should make changes to the `/2.1/` folder in the `develop` branch.

**Comment by msdolbey** [comment]
Other than that one file - this all looked good and checked out for me.

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md (hunk 1 above)
Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md]: Moved to this PR: https://github.com/mesosphere/dcos-docs-site/pull/4039

Removed in this commit: 971bbfefe97ce10cc19b89d9b726724a20f160bc

Comment (msdolbey) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md]: Solid - thanks @cbuto !

**Review by msdolbey — APPROVED** [review]
👍 

unce

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md [hunk 2]
```diff
@@ -10,10 +10,12 @@ excerpt: Deploying Kafka in a project
 
 To get started with creating and managing a Kafka Cluster in a project, you first need to deploy the [Kafka operator](../../../../../../workspaces/applications/catalog-applications/dkp-applications/kafka-operator/) and the [ZooKeeper operator](../../../../../../workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/) in the workspace where the project exists.
 
-After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
+After deploying the Kafka operator, create Kafka Clusters by applying a `KafkaCluster` custom resource **on each attached cluster** in a project's namespace. Refer to the [Kafka operator repository](https://github.com/banzaicloud/koperator/tree/master/config/samples) for examples of the custom resources and their configurations.
 
 ## Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/kafka/index.md:17]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Apply the Custom Resources to each attached cluster in a project.</p>
```

File: pages/dkp/kommander/2.2/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md [hunk 3]
```diff
@@ -10,12 +10,14 @@ excerpt: Deploying ZooKeeper in a project
...
 To get started with creating ZooKeeper clusters in your project namespace, you first need to deploy the [ZooKeeper operator](../../../../../../workspaces/applications/catalog-applications/dkp-applications/zookeeper-operator/) in the workspace where the project exists.
 
-After you deploy the ZooKeeper operator, you can create ZooKeeper Clusters by applying a `ZookeeperCluster` custom resource in a project's namespace.
+After you deploy the ZooKeeper operator, you can create ZooKeeper Clusters by applying a `ZookeeperCluster` custom resource **on each attached cluster** in a project's namespace.
 
 A [Helm chart](https://github.com/pravega/zookeeper-operator/tree/master/charts/zookeeper) exists in the ZooKeeper operator repository that can assist with deploying ZooKeeper clusters.
 
 ### Example deployment
 
+<p class="message--note"><strong>NOTE: </strong>The Custom Resources must be applied to each attached cluster in a project.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.2/projects/applications/catalog-applications/dkp-applications/custom-resources-workspace-catalog/zookeeper/index.md:19]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Apply the Custom Resources to each attached cluster in a project.</p>
```

**Review by cmastr — APPROVED** [review]

**Comment by cmastr** [comment]
Something's going on with this PR. Need to look for a merge conflict maybe.

**Comment by msdolbey** [comment]
I'll rebase to see if that works.

**Comment by msdolbey** [comment]
No conflicts now ... Git is weird ... FYI - good to rock and roll @cbuto @cmastr

**Pull request merged**